src/main/java/org/embl/mobie/io/ome/zarr/loaders/N5OMEZarrImageLoader.java

```diff
@@ -322,16 +322,20 @@ private void open() {
     private ArrayList<ViewRegistration> createViewRegistrations(int setupId, int setupTimePoints) {
         OmeZarrMultiscales multiscales = setupToMultiscale.get(setupId);
         AffineTransform3D transform = new AffineTransform3D();
-        if (multiscales.datasets[setupId].transformations != null && multiscales.datasets[setupId].transformations[0].scale != null) {
-            double[] scale = multiscales.datasets[setupId].transformations[0].scale;
-            if (scale.length > 2) {
-                transform.scale(scale[0] / 1000, scale[1] / 1000, scale[2] / 1000);
-            } else {
-                transform.scale(scale[0] / 1000, scale[1] / 1000, 1);
+        if (multiscales.datasets[setupId].coordinateTransformations != null) {
+            double[] scale = multiscales.datasets[setupId].coordinateTransformations[0].scale;
+            if (scale != null) {
+                int l = scale.length;
+                if (l > 2) {
+                    transform.scale(scale[l - 3], scale[l - 2], scale[l - 1]);
```

Comment:
ideally we would use the `axes` labels for indexing here, as scale now always has the same lenght as `axes`.
The python pseudo-code for doing this would be:
```python
axis_names = [ax["name"] for ax in axes]  # put axis names into list
spatial_axes = "xyz" if "z" in axis_names else "xy"  # set the names and order of spatial axes
scale_values = [scale[axis_names.index(ax)] for ax in spatial_axes]  # get the scalevalues for the spatial axes
```